performance_test/src/experiment_execution/analysis_result.hpp

```diff
@@ -154,14 +156,17 @@ class AnalysisResult
   const uint64_t m_num_samples_lost = {};
   const std::size_t m_total_data_received = {};
 
-  const StatisticsTracker m_latency;
```

Comment:
@deeplearningrobotics is it okay if I remove the consts from here, since I need to alter those values (i.e. variance of m_latency). Or should I add the `#ifdef PERFORMANCE_TEST_ODB_FOR_SQL_ENABLED` to only be able to change values when saving to database?